src/models/entity-contact.js

```diff
@@ -0,0 +1,38 @@
+import { UUIDV4 } from "sequelize";
+
+const entityContact = (sequelize, DataTypes) => {
```

Comment:
Since you're using this junction table here, do we still need the foreign keys on the original models?